webapp/backend/src/server.ts

```diff
@@ -0,0 +1,23 @@
+// Import libraries
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+
+// Server
+const app: express.Application = express();
+const address = "http://localhost:3000";
```

Comment:
the frontend is also being served at 3000 so wondering if that will be an issue also. Maybe we want to serve the backend at a different port to avoid any issues? I have seen some code elsewhere where they both were on the same port. let me know what you think here

Reply:
I have changed the port to 5000.